design/certified_collection_spike.md

```diff
@@ -0,0 +1,264 @@
+# Ansible Conjur Collection Certified Effort - Spike
+
+Figure out what we'll need to do to get
+[cyberark/ansible-conjur-collection](https://github.com/cyberark/ansible-conjur-collection)
+to Certified level based on our
+[community guidelines](https://github.com/cyberark/community/blob/main/Conjur/conventions/certification-levels.md#certified).
+
+CyberArk internal issue link:
+[ONYX-14258](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-14258)
+
+### What is Ansible?
+
+> Ansible is a radically simple IT automation engine that automates cloud
```

Comment:
Are the '>' intended to be there?

Reply:
Pretty sure they are, these are blockquotes and I think John is quoting lines he has pulled out of Ansible docs / articles.